tests/files_for_tests/coils.example

```diff
@@ -258,5 +258,5 @@ mirror NIL
   1.237565317338557E+01  -1.018174898724600E+01   1.933202743533675E-01   9.825067085820705E+06
   1.239173356270819E+01  -1.041975174744141E+01   5.196839261284493E-02   9.825067085820705E+06
   1.236279152161795E+01  -1.065567485505797E+01  -8.915871208808870E-02   9.825067085820705E+06
-  1.228696590129009E+01  -1.088179419180126E+01  -2.283491710711771E-01   0.000000000000000E+00 10 RotatedCurve5
+  1.228696590129009E+01  -1.088179419180126E+01   0.000000000000000E+00   0.000000000000000E+00 10 RotatedCurve5
```

Comment:
This point seems to be further away from z=0 than the preceding point - it may not affect the test, but it may be an invalid set of points???

Reply:
That's a good catch. I forgot that this file actually has two filaments in it and I modified the first coordinate in the first filament and the final coordinate in the other filament